conductor/clarisse/scripted_class/preview_ui.py

```diff
@@ -149,6 +149,8 @@ def on_write_but(self, sender, eventid):
         """
         with cu.waiting_cursor():
             self.submission.write_render_package()
+            if os.name == "nt":
```

Comment:
Can you extract this into a helper method `is_windows()`, and re-use it where we check for `os.name == "nt"`?

Reply:
done